test/kitchen/kitchen-azure.yml

```diff
@@ -220,24 +209,14 @@ suites:
       # Get the latest release agents. The upgrade recipe will take care of
       # adding the staging repo and upgrading to the latest candidate
       agent6: true
-      aptrepo: http://apt.datadoghq.com/
-      aptrepo_dist: stable
+      agent6_aptrepo: http://apt.datadoghq.com/
+      agent6_aptrepo_dist: stable
```

Comment:
There is a slight discrepancy here: for debian/ubuntu, the apt dist is `stable`, however in next lines, the yum is configured to use the `beta` channel.
It works because stable and beta are older than a pipeline build, but to be consistent it would be great to have the same channel (`beta` or `stable`) used with every repos. What do you think?

Reply:
That's true, I think we want to use the `stable` yum repo since the goal of this recipe is installing the latest stable agent and then upgrading it.